app/main.py

```diff
@@ -1 +1,62 @@
-# write your code here
+class SoftwareEngineer:
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.skills = []
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+    def __init__(self, name: str) -> None:
+        super().__init__(name=name)
+        front_skills = [
+            "JavaScript",
+            "HTML",
+            "CSS",
+        ]
+        self.skills.extend(front_skills)
+
+    def create_awesome_web_page(self) -> str:
+        print(f"{self.name} is creating a webpage...")
+        return "<h1>Hello world</h1>"
+
+
+class BackendDeveloper(SoftwareEngineer):
+    def __init__(self, name: str) -> None:
+        super().__init__(name=name)
+        back_skills = [
+            "Python",
+            "SQL",
+            "Django",
+        ]
+        self.skills.extend(back_skills)
+
+    def create_powerful_api(self) -> str:
+        print(f"{self.name} is creating an API...")
+        return "http://127.0.0.1:8000"
+
+
+class AndroidDeveloper(SoftwareEngineer):
+    def __init__(self, name: str) -> None:
+        super().__init__(name=name)
+        self.name = name
+        android_skills = [
+            "Java",
+            "Android studio",
+        ]
+        self.skills.extend(android_skills)
+
+    def create_smooth_mobile_app(self) -> str:
+        print(f"{self.name} is creating a mobile app...")
+        return "Ads every three swipes"
+
+
+class FullStackDeveloper(FrontendDeveloper, BackendDeveloper):
+    def __init__(self, name: str) -> None:
+        super().__init__(name=name)
```

Comment:
You don't really need this __init__ if it's just a super() call with the same arguments

Reply:
Still same question as here:
https://github.com/mate-academy/py-software-engineers/pull/319/files/be88e93d9e570b773a34b3936b490aa33469c698